tenant-cd/src/main/java/ai/vespa/hosted/cd/TestConfig.java

```diff
@@ -0,0 +1,101 @@
+package ai.vespa.hosted.cd;
```

Comment:
The idea is that public members of ai.vespa.hosted.cd will be accessible to the tenants, while all other packages should not be accessed by tenants?

Reply:
Yes, but this isn't really enforced yet :) There will be much moving back and forth between packages, etc..

Reply:
For this particular class, I was considering a public interface, and an implementation in the http package.

Reply:
👍 

I meant in general, not only this class.

Enforcing will be very difficult as everything ends up on the class path. But it would be great to document expected API in code, in documentation, etc.

I think we should make all internal packages more explicit, e.g. under an http.impl package.